transaction/_manager.py

```diff
@@ -215,12 +215,73 @@ def run(self, func=None, tries=3):
                 return result
 
 
-class ThreadTransactionManager(TransactionManager, threading.local):
-    """Thread-aware transaction manager.
+@implementer(ITransactionManager)
+class ThreadTransactionManager(threading.local):
+    """
+    Thread-local transaction manager.
+
+    A thread-local transaction manager can be used as a global
+    variable, but has a separate copy for each thread.
 
-    Each thread is associated with a unique transaction.
+    Advanced applications can use the `manager` attribute to get a
+    wrapped TransactionManager to allow cross-thread calls for
+    graceful shutdown of data managers.
     """
 
+    def __init__(self):
+        self.manager = TransactionManager()
+
+    @property
+    def explicit(self):
+        return self.manager.explicit
+
+    @explicit.setter
+    def explicit(self, v):
+        self.manager.explicit = v
+
+    def begin(self):
+        return self.manager.begin()
+
+    def get(self):
+        return self.manager.get()
+
+    __enter__ = get
```

Comment:
This makes assumptions about TransactionManager's implementation that may be true today, but might change in the future?

Why not forward to `self.manager.__enter__()`?

Reply:
Fair enough.  I'll change.

Reply:
Other than the dunder methods, could this be implemented with `__getattr__` and `__setattr__` forwarding to `self.manager`? That's less explicit, but much shorter. Or does that break interface validation?

Reply:
done